recipes/libhugetlbfs/build.sh

```diff
@@ -0,0 +1,18 @@
+if [[ ${ARCH} == "64" ]]; then
+    export ARCH=x86_64
+fi
+
+# Remove the static library from being installed
+sed -i 's/libhugetlbfs.a//g' Makefile
```

Comment:
Should we also test to see this is removed?

Reply:
done.